src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslMinPortSpeedSpec.groovy

```diff
@@ -38,7 +38,9 @@ class IslMinPortSpeedSpec extends HealthCheckSpecification {
         when: "Replug one end of the connected link to the destination switch(isl.srcSwitchId -> newDst.srcSwitchId)"
         def newIsl = islUtils.replug(isl, false, newDst, true, true)
 
-        islUtils.waitForIslStatus([newIsl, newIsl.reversed], DISCOVERED)
+        Wrappers.wait(discoverySlowPollInterval + WAIT_OFFSET) {
+            [newIsl, newIsl.reversed].each { assert islUtils.getIslInfo(it).get().state == DISCOVERED }
```

Comment:
it is cleaner to use `northbound.getLink` here to ask for 1 certain isl. `islUtils.getIslInfo(it)` will dump all isls twice